src/js/components/Grommet/Grommet.js

```diff
@@ -93,15 +95,23 @@ const Grommet = forwardRef((props, ref) => {
     deviceResponsive(userAgent, theme) ||
     theme.global.deviceBreakpoints.tablet;
 
+  const grommetRef = useForwardedRef(ref);
+  // When toggling aria-hidden values, we only want to affect elements
+  // in the DOM that come from Grommet, so we track those elemnts in this
+  // context value. See FocusedContainer.js
+  const bodyChildrenFromGrommet = [grommetRef.current];
```

Comment:
What if I have nested `<Grommet />` elements?

Reply:
Hmm that's a good point. My understanding is that Context is contained, so I'm not sure if you can have an outer/inner context inherit the value of the other -- do you know?

Otherwise, we may need to rely more explicitly on a DOM attribute like "data-g-id" or something that we could scan the DOM for. What do you think?

Reply:
I think we have to avoid scanning the DOM. I was thinking that we just keep the additive nature, so a nested Grommet just adds another root to the array started by the outermost one.